springwolf-plugins/springwolf-spring-cloud-rabbit-binder-plugin/build.gradle

```diff
@@ -0,0 +1,145 @@
+plugins {
+    id 'java-library'
+    id 'maven'
+    id 'signing'
+    id 'maven-publish'
+
+    id 'org.springframework.boot' version '2.6.2'
+    id 'io.spring.dependency-management' version '1.0.6.RELEASE'
+}
+
+group 'io.github.springwolf'
+version '0.3.1'
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    api project(':springwolf-core')
+
+    implementation 'org.springframework:spring-web'
+    implementation 'org.springframework.cloud:spring-cloud-stream-binder-rabbit:3.2.4'
```

Comment:
I think that we should avoid specifying a version for the spring dependencies

Reply:
I commented above, but I can't get it to build without specifying a version number